backend/routers/admin_analytics.py

```
@@ -322,32 +338,22 @@ async def get_ai_pricing_recommendations(request: Request, db=Depends(get_db)):
-            "recommendations": [],
-            "summary": {
-                "total_recommendations": 0,
-                "total_expected_impact": 0,
-                "high_priority_count": 0,
-                "average_confidence": 0,
-                "recommendations_by_type": {}
-            },
-            "last_updated": datetime.now().isoformat(),
-            "error": "தரவு ஏற்ற முடியவில்லை"
-        } 
+        raise HTTPException(status_code=500, detail=f"Internal Server Error: {e}")
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)